docs/v1.1.0/schema.md

```diff
@@ -0,0 +1,216 @@
+|                     |                      | Corresponding author status | corresponding_author_status  | BOOLEAN                                 |  --  |   --    |                     OPTIONAL                      | N/A                                   | Indicating whether an author is the corresponding author (YES or NO)                                                                                                                                     |
+|                     |                      | Email                       | email                        | STRING                                  |  --  |   --    | MUST or OPTIONAL<sup>**[1](#author_email)**</sup> | N/A                                   | Email address for each author                                                                                                                                                                            |
+|                     |                      | Affiliation name            | affiliation_name             | STRING                                  |  --  |   --    |                    RECOMMENDED                    | ROR                                   | Name of the institution an author is affiliated with. Sometimes, one author may have multiple affiliations                                                                                               |
+|                     |                      | Affiliation address         | affiliation_address          | STRING                                  |  --  |   --    |                     OPTIONAL                      | N/A                                   | Address of the institution an author is affiliated with                                                                                                                                                  |
+|                     |                      | Affiliation identifier      | affiliation_identifier       | STRING                                  |  --  |   --    |                    RECOMMENDED                    | ROR                                   | A unique identifier assigned to the affiliated institution by The Research Organization Registry (ROR)                                                                                                   |
+|                     |                      | Author list order           | order                        | INTEGER                                 |  --  |   --    |                       MUST                        | N/A                                   | The order in which the author appears in the publication                                                                                                                                                 |
```

Comment:
The entire Author section is actually not a dictionary, but a list of Dictionaries of type "DatasetAuthor". We should reflect this in the documentation.

Reply:
I don't know if we want to accept an explicit list order vs making that implied by the order that the authors are listed in the metadata file. Personally I'd prefer to make the ordering in the file significant.